apkpure/apkpure.py

```diff
@@ -63,13 +63,14 @@ def get_package_data() -> dict:
             
             if not package_data.get('class', ''):
                 package_data = html_element.find("a", class_="is-download")
-            
+            package_extension = package_data.get("data-dt-apk-type", 'Uknown')
```

Comment:
When the app is an XAPK we need to get this information to proceed, like to get the download link based on the file format